devops/containers/ubuntu2004_preinstalled.Dockerfile

```diff
@@ -5,7 +5,8 @@ FROM $base_image:$base_tag
 
 COPY scripts/drivers_entrypoint.sh /drivers_entrypoint.sh
 RUN mkdir -p /opt/sycl
-ADD llvm_sycl.tar.xz /opt/sycl
+RUN tar -I 'zstd -d' -xf llvm_sycl.tar.zst -C /tmp/llvm_sycl
+ADD /tmp/llvm_sycl /opt/sycl
```

Comment:
```suggestion
ADD llvm_sycl.tar.zst /tmp
RUN tar -I 'zstd -d' -xf /tmp/llvm_sycl.tar.zst -C /opt/sycl && rm llvm_sycl.tar.zst
```

Reply:
Should this be `rm /tmp/llvm_sycl.tar.zst`?